src/components/TableBody.js

```diff
@@ -283,7 +283,7 @@ class TableBody extends React.Component {
               colIndex={0}
               rowIndex={0}>
               <Typography variant="body1" className={classes.emptyTitle}>
-                <strong>{options.textLabels.body.noMatch}</strong>
+                {options.textLabels.body.noMatch}
```

Comment:
Is there a reason for removing the strong(bold)?

It's different.
https://codesandbox.io/s/material-demo-buwx0?file=/demo.js

Reply:
oh.. https://github.com/gregnb/mui-datatables/pull/1154

I found it here.

that's right 
It seems right to remove strong tag.